lib/webpacker/dev_server.rb

```diff
@@ -0,0 +1,54 @@
+# Same convention as manifest/configuration.rb
+# Loads webpacker configuration from config/webpacker.yml
+
+class Webpacker::DevServer < Webpacker::Configuration
```

Comment:
@gauravtiwari, I think you'll need to comment here as you suggested this. I'm happy to merge these together.